### PR TITLE
Add post index

### PR DESCRIPTION
**Pull request by CamBrown101**
- Add post index feature and tests

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,4 +1,9 @@
 class PostsController < ApplicationController
+    def index
+      @posts = Post.all
+      render :index
```
Comment (aarondufall) [inline app/controllers/posts_controller.rb:4]: I don't think you will need this line for it to default to the index template. That being said a case could be made to have it explicit in the code. What's important is that you and your team are consistent with these things.

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<h1>All Posts</h1>
+
+<table>
```
Comment (aarondufall) [inline app/views/posts/index.html.erb:3]: You may want to consider using `<thead>` and `<tbody>` tags

File: spec/controllers/post_controller_spec.rb [hunk 3]
```diff
@@ -1,6 +1,23 @@
 require 'rails_helper'
 
 RSpec.describe PostsController, type: :controller do
+    describe "#index" do
```
Comment (aarondufall) [inline spec/controllers/post_controller_spec.rb:4]: 👍

File: spec/features/posts_spec.rb [hunk 4]
```diff
@@ -16,4 +16,14 @@
     click_button "Create Post"
     expect(page).to have_content "can't be blank"
   end
-end
+
+  scenario "user can view all posts" do
+    post_one = FactoryBot.create(:post)
+    post_two = FactoryBot.create(:post, title: "foo", body: "bar" )
```
Comment (aarondufall) [inline spec/features/posts_spec.rb:22]: If you add Faker having different data for each of your factories would be automatic. Check out the fake data section in your testing book

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (CamBrown101) [reply app/controllers/posts_controller.rb:4]: Do you mean the render line?

**Pull request merged**